packages/js/settings-editor/src/layout.tsx

```diff
@@ -22,6 +20,17 @@ import {
 	// @ts-expect-error missing type.
 	__unstableAnimatePresence as AnimatePresence,
 } from '@wordpress/components';
+// eslint-disable-next-line @woocommerce/dependency-group
```

Comment:
This's looking good. We can also consider disabling the dependency group lint rule for these imports and re-enable it after the import.

```js
/* eslint-disable @woocommerce/dependency-group */

// import...

/* eslint-enable @woocommerce/dependency-group */
```

Reply:
👍🏽  Done in https://github.com/woocommerce/woocommerce/pull/52655/commits/7e5792fa0898f9f585fd286cb5c9017307f97c87